configure

```diff
@@ -1901,7 +1913,19 @@ s!@docdir@!${docdir}!g
 s!@htmldir@!${htmldir}!g
 s!@default_loadpath@!${default_loadpath}!g
 s!@enable_nls@!${enable_nls}!g
+s!@enable_doc@!${enable_doc}!g
+"
+if [ "$enable_doc" = "true" ]; then
+    sed_subst_cmd="
+$sed_subst_cmd
+s!@doc_mandir@! \$\(DESTDIR\)\$\(mandir\)!g
 "
+else
+    sed_subst_cmd="
+$sed_subst_cmd
+s!@doc_mandir@!!g
+"
+fi
```

Comment:
Please define an intermediate shell variable instead of directly manipulating `sed_subst_cmd`. That's how other make variables are computed.

```sh
if ${enable_doc}
then
    doc_mandir=' $(DESTDIR)$(mandir)'
else
    doc_mandir=
fi
```

```
s!@doc_mandir@!${doc_mandir}!g
```